index.html

```diff
@@ -11761,7 +11765,7 @@ <h1>JSON-LD Grammar</h1>
   <section id="terms" class="normative">
     <h2>Terms</h2>
 
-    <p>A <a>term</a> is a short-hand <a>string</a> that expands
+    <p>A <a>term</a> is a short-hand <a>string</a> that MAY expand
```

Comment:
```suggestion
    <p>A <a>term</a> is a short-hand <a>string</a> that SHOULD expand
```

Reply:
"MAY" was previously used in section 1.4, so I reflected it there as is.

I'm not sure that either MAY or SHOULD are a good match here. Whether a term *can* be expanded does not depends on the implementation, and an implementation MUST expand in some circumstances (e.g. expand algorithm), and MUST NOT expand it in other (e.g. compact algorithm).

I think it is best to revert this line to its original version, or keep "may" but in lowercase?

Reply:
Okay, we'll revert, then.